pdfannots.py

```diff
@@ -367,18 +369,17 @@ def format_annot(self, annot, extra=None):
         # comment, and the text contains no embedded full stops or quotes, then
         # we'll just put quotation marks around the text and merge the two into
         # a single paragraph.
-        if (text and len(text) == 1 and len(text[0].split()) <= 10 # words
-            and all([x not in text[0] for x in ['"', '. ']])
```

Comment:
I removed this line since it was interfering with the length option which already gives increased control over annotations' output format. If you prefer, I can also add another optional argument for controlling which characters causes annotations to be outputted in short form and set the default to the list from this line.

Reply:
This is the one part I'm uncomfortable about losing, actually. I've (selfishly) set the style to match what I'm comfortable with appearing in a review, and I find that if a comment is short but has embedded punctuation then it's going to look out of place in the single-line format where the text and comment aren't as clearly separated.